internal/cmd/server.go

```diff
@@ -133,8 +134,9 @@ The kernel is used to run long running processes like shells and interacting wit
 	cmd.Flags().StringVarP(&addr, "address", "a", defaultAddr, "Address to create unix (unix:///path/to/socket) or IP socket (localhost:7890)")
 	cmd.Flags().BoolVar(&devMode, "dev", false, "Enable development mode")
 	cmd.Flags().BoolVar(&enableRunner, "runner", true, "Enable runner service (legacy, defaults to true)")
+	cmd.Flags().BoolVar(&enableAILogs, "ai-logs", false, "Enable logs to support training an AI")
 	cmd.Flags().StringVar(&tlsDir, "tls", defaultTLSDir, "Directory in which to generate TLS certificates & use for all incoming and outgoing messages")
-
+	cmd.Flags().StringVar(&configDir, configDirF, GetDefaultConfigHome(), "If ai logs is enabled logs will be written to ${config-dir}/logs")
```

Comment:
Any reason not to reuse `fLogFilePath` as base? The config home is a bit weird on `darwin` (and frankly, tentative). Also, as part of the `beta` commands (incomplete alpha) we are migrating config away from a central location to a `runme.yaml` that resides inside a project/repository.

Reply:
So I think we can break this down into separate questions

1. What's the default location for logs? e.g. is it ${CONFIGDIR}/logs?
2. What flags control the location of the logs?

Regarding the first question.

* I don't think we want the default value to be a temporary directory because the logs are too valuable to be lost when the temporary directory gets cleaned up

> we are migrating config away from a central location to a runme.yaml that resides inside a project/repository

Are you suggesting the logs get stored relative to the project/repo specific runme.yaml? Or is this just an indication that the runme user configuration is going away in favor of per project/repository configuration.

Either way; I think of log location as being configured per user not per project/repository. Which implies we need some per user directory.

> The config home is a bit weird on darwin

What about ${HOME}/.runme for  runme user configuration and ${HOME}/.runme/logs

Two; I think once we agree on where logs should be stored we can decide if it makes sense to change fLogFilePath to use it and if so use it to compute the location of the AI Logs.

Reply:
I see your point here, especially that it's a `user` vs a `server` log. That really answers both questions 1. & 2. for me. Here's my suggestion, we roll with your code change as is, however, we rename `GetDefaultConfigHome` to `GetUserConfigHome` to make the difference in base directories more apparent and future proof @jlewi?

Ideally `GetDefaultConfigHome` / `GetUserConfigHome` would actually be rooted in the home directory (on all platforms) but we can fix that at a later time. I'll create a bug ticket for it.

Reply:
Done.